lib/SearchResultsList/SearchResultsList.js

```diff
@@ -84,17 +72,11 @@ const SearchResultsList = ({
   hidePageIndices,
   hasNextPage,
   hasPrevPage,
+  onRenderHeadingRef,
```

Comment:
Better to name it `renderHeadingRef` - `on` prefix is used for callback functions

Reply:
Thanks, fixed